.github/workflows/deploy-elements-canary.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy @carbon/ibmdotcom-elements storybook to IBM Cloud (Canary)
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Install dependencies
+        run: yarn install --offline
+      - name: Build project
+        run: yarn build
```

Comment:
Later on we would probably need to add environment variables for the CORS_PROXY, kaltura partner ID, etc. Should we go ahead and add them now or wait til we roll in the components that need them?